Tools/autotest/quadplane.py

```diff
@@ -103,6 +103,40 @@ def get_disarm_delay(self):
     def set_autodisarm_delay(self, delay):
         self.set_parameter("LAND_DISARMDELAY", delay)
 
+    def test_motor_mask(self):
+        """Check operation of output_motor_mask"""
+        if not(int(self.get_parameter('Q_TILT_MASK')) & 1):
```

Comment:
Does setting this really require a reboot?  The parameter is not marked as such.

Reply:
being paranoid here: wanted to guarantee that the copter state hasn't been touched

Reply:
It would be nice to have a fresh vehicle before each test.  However, it's kind of too expensive time-wise to do so.

We do do some work to try to give each test a vehicle in set state - disarmed, on ground, with default parameters.